src/fsharp/FSharp.Compiler.Interactive.Settings/FSharp.Compiler.Interactive.Settings.fsproj

```diff
@@ -15,9 +15,9 @@
     <FileAlignment>512</FileAlignment>
   </PropertyGroup>
   <ItemGroup>
-      <FsSrGen Include="..\FSInteractiveSettings.txt">
```

Comment:
Per contribution guidelines, pure cleanup should be done separately. This just adds needlessly to the diff.

> Code cleanup which is unrelated to a bug fix or feature should generally be made separate to other checkins where possible.